### DISCUSSION TITLE
How to prevent users to see all PCs/Clients which ar not assigned to a user

### DISCUSSION THREAD

**Discussion by AWHarvard2** [rustdesk/rustdesk-server-pro#246] [General]
We have for example two user groups:

1 Group: Administrators, checkbox admins checked
2 Group: Lab users (no admins)

How to prevent users in the Lab group from seeing PCs which are not assigned to a user or group?
At the moment users in that group are able to see deviced added to the Lab group but **also** deviced which are not assigned to any group/user. I guess this schould be able to admins only, otherwise that would break the idea of restricting access by groups to us due to installing hundreds of new clients in bulk.

**Comment by rustdesk** [comment]
There shows all devices the user have permission to control.

**Reply by AWHarvard2** [reply]
But why does it show all devices currently not linked to any group to a user which should be restricted only to the lab group?

**Reply by rustdesk** [reply]
https://github.com/rustdesk/rustdesk-server-pro/discussions/175#discussioncomment-7829424

**Reply by AWHarvard2** [reply]
No way to prevent these as we have only 10 rustdesk users (admins) and about 100 PCs?!

**Reply by rustdesk** [reply]
Is this https://github.com/rustdesk/rustdesk-server-pro/discussions/214 your expected?

**Reply by AWHarvard2** [reply]
Excaly #214 would be what we need.

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk-server-pro/releases/tag/1.3.5

**Comment by ZetaWaves** [comment]
Is there a way to remove the 'Groups' list from the client? It is removed when you disable account logins, but why can't there be a way to disable the groups list, but also let the user log in.

For example, I did this, but I believe there's a better way to remove it entirely from the build like the disable account login does:

advanced settings > overrides:
`peer-tab-visible=[true,true,true,true,false]`

**Reply by rustdesk** [reply]
I have seen https://github.com/rustdesk/rustdesk-server-pro/issues/250, be patient please. We may add this if I see more Pro users request it.
